core/src/main/scala/com/stripe/dagon/HMap.scala

```diff
@@ -81,6 +81,9 @@ final class HMap[K[_], V[_]](protected val map: Map[K[_], V[_]]) extends Seriali
     val fnAny = f.toFunction[Any].andThen(_.iterator)
     map.iterator.asInstanceOf[Iterator[(K[Any], V[Any])]].flatMap(fnAny).toStream
   }
+
+  def mapValues[V1[_]](f: FunctionK[V, V1]): HMap[K, V1] =
+    HMap.from[K, V1](map.mapValues(f(_)))
```

Comment:
Actually one request, mapValues is lazy on maps and this often burns people. Can we make this strict? `map.iterator.map { case (k, v....` or something?

Reply:
sure!